docs/Tags.md

```diff
@@ -73,19 +81,58 @@ FT.SEARCH myIndex "@cities:{ New York | Los Angeles | Barcelona }"
 But the next query will return all people who have visited **all three cities**:
 
 ```
-@cities:{ New York } @cities:{Los Angeles} @cities:{ Barcelona }
+FT.SEARCH myIndex "@cities:{ New York } @cities:{Los Angeles} @cities:{ Barcelona }"
+```
+
+## Including punctuation in tags
```

Comment:
Combining treatment of tags that include spaces with tags that include punctuation might make sense logically, but the material is easier to read separately and easier to find if a person is looking for one or the other use case.